test/src/tokenizer.spec.ts

```diff
@@ -0,0 +1,44 @@
+import expect from 'expect';
+import {tokenize} from 'puppeteer-core/internal/injected/PSelectorTokenizer.js';
+
+describe('PSelectorTokenizer', () => {
```

Comment:
Let's add some parsing tests for child, sibling, general sibling, adjacent sibling and pseudo elements?